CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- [Correcting incorrect closing double quotes in pagination Nunjucks](https://github.com/alphagov/govuk-frontend/pull/3156)
```

Comment:
Massive nitpick but we try to put fixes under the '## Fixes` heading below. Would you mind moving this to a new line under line 30? Happy to do this for you if you prefer.